frontend/src/features/map_planning/components/BaseStage.tsx

```diff
@@ -229,19 +233,14 @@ export const BaseStage = ({
       >
         {children}
         <Layer>
-          <Label visible={false} ref={tooltipRef} scaleX={1 / stage.scale} scaleY={1 / stage.scale}>
-            <Tag fill="black" />
-            <Text fill="white" fontSize={24} padding={6} />
-          </Label>
-          <Rect
-            x={selectionRectAttrs.x}
-            y={selectionRectAttrs.y}
-            width={selectionRectAttrs.width}
-            height={selectionRectAttrs.height}
-            fill={'blue'}
-            visible={selectionRectAttrs.isVisible}
-            opacity={0.2}
-            name="selectionRect"
+          {/* Tooltip */}
+          <MapLabel
+            content={tooltipContent}
+            visible={tooltipContent !== ''}
+            scaleX={3 / stage.scale}
+            scaleY={3 / stage.scale}
```

Comment:
It looks to me that the mouse-over text is now bigger?

Reply:
Yes I had to make it larger because the tooltip would be kind of tiny otherwise.